### PR TITLE
V2 release notes

### PR DESCRIPTION
**Pull request by lottamus**
https://deploy-preview-1592--stoplightio.netlify.app/blog/stoplight-v2-0-0-release/

### CODE REVIEW

File: netlify/blog-posts/platform-and-studio-v2.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+path: /blog/stoplight-v2-0-0-release
+tags:
+  - blog
+publishedDate: 2020-09-02T12:26:13.173Z
+author: Phil Sturgeon
+title: Stoplight v2.0 Release
+subtitle: Studio and Platform
```
Comment (philsturgeon) [inline netlify/blog-posts/platform-and-studio-v2.md:8]:
```suggestion
subtitle: Workspaces, Automatic Publishing, Theming, and multiple branches in documentation.
```

File: netlify/blog-posts/platform-and-studio-v2.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+Last month we announced [Stoplight Platform](https://stoplight.io/blog/stoplight-platform/) to the world, a revolutionary ecosystem that lets you handle API Design, Documentation, and API Mocking. Unlike other tooling, all this functionality, and the API descriptions and documentation can live in your existing version control system, along with the source code. The new approach handles all this whilst keeping things easy enough for your whole team to use, not just the techie folks.
+
+Now that we've had some time to really polish things, we're ready to release major versions for two parts of our ecosystem:
+
+- [Stoplight Studio Desktop v2.0](#studio-desktop-v2-0)
+- [Stoplight Platform v2.0](#platform-v2-0)
+
+For those of you new to [Studio Desktop](/studio), it's is a standalone application for Windows, macOS, and Linux, which allows you to edit and manage OpenAPI, JSON Schema, AsyncAPI, and Markdown documents locally. Studio Desktop works with files on your computer, has built-in Git functionality, runs local mocking servers for you, and it's totally free. You can use it with any Git repository or local directory you want, whether it has anything to do with Stoplight or not.
```
Comment (philsturgeon) [inline netlify/blog-posts/platform-and-studio-v2.md:36]:
```suggestion
For those of you new to [Studio Desktop](/studio), it's is a standalone application for Windows, macOS, and Linux, which allows you to edit and manage OpenAPI, JSON Schema, AsyncAPI, and Markdown documents locally. Studio Desktop works with files on your computer, has built-in Git functionality, runs local mocking servers for you, and can be [downloaded](https://github.com/stoplightio/studio/releases) and used totally free. You can use it with any Git repository or local directory you want, whether it has anything to do with Stoplight or not. 
```

File: netlify/blog-posts/platform-and-studio-v2.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  - blog
+publishedDate: 2020-09-02T12:26:13.173Z
+author: Phil Sturgeon
+title: Stoplight v2.0 Release
+subtitle: Studio and Platform
+listSubtitle: Studio and Platform v2.0 Releases | Stoplight API Intersection
+image: /images/changelog-stock.jpg
+includeToc: true
+actionBar:
+  ctas: []
```
Comment (philsturgeon) [inline netlify/blog-posts/platform-and-studio-v2.md:13]:
```suggestion
```

File: netlify/blog-posts/platform-and-studio-v2.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+A powerful full-text search tool for exploring all of the API design assets in your Workspace. The new hierarchical view allows you to quickly traverse the contents of your API projects.
+
+![](/images/blog/studio-and-platform-v2-0/explorer.png)
+
+Check out some background on [Explorer](https://meta.stoplight.io/docs/platform/5.-governance/bb.exploring-your-api-projects.md).
+
+
+### Theming and Branding
+
+Pick one of the preset themes or provide your own colors to fully customize your workspace's branding!
```
Comment (lottamus) [inline netlify/blog-posts/platform-and-studio-v2.md:85]: @philsturgeon I'll update this to mention logo, landing page, and custom domains too

File: netlify/blog-posts/platform-and-studio-v2.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+![](/images/blog/studio-and-platform-v2-0/theming.gif)
+
+### Expanded Git Integrations
+
+Stoplight supports Team Foundation Server 2018 (v4.1), which has since been renamed to [Azure DevOps Server](https://docs.microsoft.com/en-us/azure/devops/server/tfs-is-now-azure-devops-server?view=azure-devops).
```
Comment (lottamus) [inline netlify/blog-posts/platform-and-studio-v2.md:91]: @philsturgeon I'm thinking about re-working this to mention all of the VCS integrations we support and that ADS is the most recent addition

File: netlify/blog-posts/platform-and-studio-v2.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+Workspaces are a handy way to group API projects across your organization, to promote visibility, and reuse of artifacts. We at Stoplight recommend making a single workspace for your organization and using roles and permissions to effectively manage your workflows.
+
+![](/images/v2-meta-workspace.png)
+
+Read more about [workspaces](https://meta.stoplight.io/docs/platform/2.-workspaces/a.creating-a-workspace.md).
+
+### Automatic Publishing
+
+In the past, to update your docs, somebody would have needed to pull changes in Studio Desktop or Studio Web, and manually click a publish button. Now that manual step is no longer needed. Webhooks from GitHub, GitLab, and BitBucket can all be used to automatically update your documentation, Explorer, and mock servers, and even works with multiple branches.
```
Comment (lottamus) [inline netlify/blog-posts/platform-and-studio-v2.md:56]: Might we worth mentioning CLI publish too even though that was an option before? I don't want to confuse people that might think webhooks are the only option

File: netlify/blog-posts/platform-and-studio-v2.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+### Automatic Publishing
+
+With Publishing now handled in Stoplight Platform, there's no need for the Publish button anymore, so we've gone ahead and chucked that out to clear up the UI for you.
+
+The new docs are published to a new and improved URL for Stoplight.io users, so take a look at how [publishing works in v1 vs v2](https://meta.stoplight.io/docs/studio/docs/Documentation/06-publishing.md).
+
+### Supports on-premise VCS providers
+
+In the v1, all Git requests were proxied through the Stoplight API, which required you to login to your Stoplight account and was limited to Github. Now you can use personal access tokens to authenticate with any VCS provider.
```
Comment (lottamus) [inline netlify/blog-posts/platform-and-studio-v2.md:105]: Do you think we need anymore info on here about personal access tokens? Or a screenshot of Studio where you add your token?

File: netlify/blog-posts/platform-and-studio-v2.md (hunk 6 above)
Comment (philsturgeon) [reply netlify/blog-posts/platform-and-studio-v2.md:56]: Yep, I'll take care of this when you're done.

File: netlify/blog-posts/platform-and-studio-v2.md (hunk 7 above)
Comment (philsturgeon) [reply netlify/blog-posts/platform-and-studio-v2.md:105]: Ideally we'd link them to the docs, which I'll take care of now. https://github.com/stoplightio/platform-docs/issues/83

**Comment by paulatulis** [comment]
Is there a reason netlify/blog-posts/10-000-apis-under-the-sea.md and netlify/blog-posts/all-your-semantics-belong-to-us.md were deleted?

**Comment by lottamus** [comment]
> Is there a reason netlify/blog-posts/10-000-apis-under-the-sea.md and netlify/blog-posts/all-your-semantics-belong-to-us.md were deleted?

@philsturgeon did you mean to delete these or should we add them back?

**Comment by philsturgeon** [comment]
Yeah I’ve been clearing out some older posts as I go. Let’s keep em gone.

**Review by philsturgeon — APPROVED** [review]

**Review by rossmcdonald — APPROVED** [review]
Looks good!

**Pull request merged**